markdown_stoplight/use-cases.md

```diff
@@ -1,128 +1,157 @@
 # Use cases
-You plan an integration with ePages? That’s great! Here are some use cases we can think of:
+You plan an integration with ePages?
+That’s great!
+Here are some use cases we can think of:
 
-### Integrate a Payment Solution
+### Payment solution
 
-Enable merchants to offer more payment options by integrating your own payment solution. Create your own payment method so that merchants can later activate it for their shops. This way, a merchant's customers will find a more diverse set of possibilities to pay for their purchase.</br>
+Enable merchants to offer more payment options by integrating your own payment solution.
+Create your own payment method so that merchants can later activate it for their shops.
+This way, a merchant's customers will find a more diverse set of possibilities to pay for their purchase.</br>
```

Comment:
Nothing new, but: a merchant's customers --> Either make it plural or singular

Reply:
👍